src/main/java/org/madhuri/app/message/model/Channel.java

```diff
@@ -1,66 +1,93 @@
 package org.madhuri.app.message.model;
 
+import java.util.ArrayList;
 import java.util.Date;
+import java.util.List;
+
+import javax.persistence.CascadeType;
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
 import javax.persistence.Table;
+import jakarta.json.bind.annotation.JsonbProperty;
 
 @Entity
 @Table(name = "channel")
 public class Channel {
 
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@JsonbProperty("id")
 	@Column(name = "id")
 	private Long id;
 
-	@Column(name = "name")
-	private String name;
+	
+	@Column(name = "channel_name")
+	@JsonbProperty("channelName")
+	private String channelName;
 
 	@Column(name = "admin_id")
+	@JsonbProperty("adminId")
 	private Long adminId;
 
 	@Column(name = "welcome_message")
+	@JsonbProperty("welcomeMessage")
 	private String welcomeMessage;
 
 	@Column(name = "created_at")
+	@JsonbProperty("createdAt")
 	private Date createdAt;
 
 	@Column(name = "created_by")
+	@JsonbProperty("createdBy")
 	private Long createdBy;
 
 	@Column(name = "updated_at")
 	private Date updatedAt;
 
 	@Column(name = "updated_by")
+	@JsonbProperty("updatedBy")
 	private Long updatedBy;
 
+	@ManyToMany(cascade=CascadeType.ALL,fetch=FetchType.EAGER)
```

Comment:
eager fetching will fetch all the users in this channel everytime you fetch the channel details, the API performance will be very bad.